roles/ceph-common/defaults/main.yml

```diff
@@ -73,10 +73,9 @@ ntp_service_enabled: true
 # install source or architecture.
 debian_ceph_packages:
   - ceph
-  - ceph-common    #|
-  - ceph-fs-common #|--> yes, they are already all dependencies from 'ceph'
-  - ceph-fuse      #|--> however while proceding to rolling upgrades and the 'ceph' package upgrade
-                   #|--> they don't get update so we need to force them
+  - ceph-common    #|--> yes, they are already all dependencies from 'ceph'
```

Comment:
Yes please see the commit comments for 7ae918620355125cbeb8419005c19885dbd3bc3e -- the package was removed earlier, however the current master has ugly formatting for the comments in ceph-common, and also the package removal was never synced to `group_vars/all.yml.sample` so if you run `generate_group_vars_sample.sh` on a clean checkout from master currently it will have this change. This commit fixes the comment formatting and brings the group_vars sample back into sync with ceph-common defaults.